examples/htlc-pyteal-ts/scripts/withdraw/htlc-withdraw.ts

```diff
@@ -34,12 +34,16 @@ async function run(
 		args: [algob.convert.stringToBytes(wrongSecret)],
 		payFlags: { totalFee: 1000 },
 	};
-	// Transaction Fails : as wrong secret value is used
-	await WithdrawExecuteTx(deployer, txnParams);
+	// Transaction Fails : as wrong secret value is used 
+	// await withdrawExecuteTx(deployer, txnParams); (uncomment to test)
```

Comment:
lets leave both cases and just to try catch and handle them properly

Reply:
but it will throw error, and therefore the second txn will not run, so we need to comment it

Reply:
if we use try catch it will

Reply:
I did with try catch but once we use throw error it stops the process, i guess instead of throwing the error in incorrect tx we can log it but in correct one we can throw it